src/components/header/header.tsx

```diff
@@ -0,0 +1,24 @@
+import SearchForm from '../searchForm/searchForm';
+import GenreSelect from '../genreSelect/genreSelect';
+const Header = (props:GenreSelectProps) => {
+
+    const handleSearch = (e: React.FormEvent<HTMLFormElement>)=> {
+        e.preventDefault();
+          console.log(e);    
+        console.log("Search is Working !!");
+    
```

Comment:
Code formating! Indentations, empty lines.
Please remove the sevice console logs (console.log(e))